ergast_py/type_constructor.py

```diff
@@ -224,6 +225,16 @@ def construct_race(self, race: dict) -> Race:
             laps=self.construct_laps(race["Laps"]),
         )
 
+    def construct_time(self, time: dict) -> Time:
```

Comment:
As Time is now a model we want to be able to populate any missing fields etc. I think it may be overkill but it's how every other model ensures all expected fields are present at least.

That would be a new function (i.e. `self._populate_missing_time`) and a new entry into `expected.py` for the time object.

Reply:
In `expected.py` file, what should be the expected type for `millis`? I have kept `datetime.time` for now.